### DISCUSSION TITLE
How to customize block with lsp info

### DISCUSSION THREAD

**Discussion by honyani** [folke/noice.nvim#320] [Q&A]
How to change color of this block? I have transparent background in my nvim how to make it similiar?

<img width="494" alt="Снимок экрана 2023-01-16 в 22 23 28" src="https://user-images.githubusercontent.com/118450997/212752692-9748ef60-a3c8-4494-98a1-759b962053f2.png">

**Comment by gennaro-tedesco** [comment]
Have you tried
```lua
...
views = {
	mini = { win_options = { winblend = 100 } },
},
```
See [here](https://github.com/folke/noice.nvim/issues/226) for reference.

**Reply by honyani** [reply]
@gennaro-tedesco thank you! Everything works, only instead of "winbled = 100" need to write 0 value
